tests/cases/fourslash/completionListInImportClause05.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: app.ts
+//// import * as A from "[|/*1*/|]";
+
+// @Filename: /node_modules/@types/a__b/index.d.ts
+////declare module "@e/f" { function fun(): string; }
+
+// @Filename: /node_modules/@types/c__d/index.d.ts
+////export declare let x: number;
+
+// NOTE: When performing completion, the "current directory" appears to be "/",

```

Comment:
I think this may be a correct behavior -- `host.getCurrentDirectory()` returns `process.cwd();`, not necessarily the root of your project.

Reply:
I've refined the comment.